### DISCUSSION TITLE
Connecting Multiple Servers

### DISCUSSION THREAD

**Discussion by rxerium** [telekom-security/tpotce#1446] [Q&A]
Hi there,

I'm going to be ordering 20-30 servers on the internet all of which I will be setting up t-pot on. Is it possible to connect all of these to one main instance where I can track/manage threats/attacks against any of the instances I own instead of logging into each of the nodes?

Also, if I am to do this, will I need to set up a full instance of t-pot on each of the additional "nodes" or will 2GB RAM/1CPU core suffice?

Thanks all!

Rishi

**Comment by dmille6** [comment]
you want a "hive" install.. thats exactly what its designed for.. start small.. ease into this.. so you see how much data you're getting.. and how to manage it.

**Comment by rxerium** [comment]
Ahh I see, currently I have a "STANDARD" install, I'll reinstall the OS and set it to HIVE, unless there is an easier way of switching? 

Presuming the additional servers I'll have to set to HIVE_SENSOR?

Do you know how I can connect the HIVE_SENSORs to the HIVE? Would you be able to provide a link to documentation on this?
